examples/slurm/submit_multicpu.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash -l
+
+#SBATCH --job-name=multicpu
+#SBATCH --nodes=2                       # number of Nodes
+#SBATCH --ntasks-per-node=1             # number of MP tasks
+#SBATCH --exclusive
+#SBATCH --output=output/torch_%A.log
+#SBATCH --cpus-per-task=56
+
+######################
+### Set enviroment ###
+######################
+source activateEnviroment.sh
```

Comment:
```suggestion
source activateEnvironment.sh
```

😂  Just realized there's a typo here and that's why my env file wasn't being found. The same typo is in all of the submit_*.sh script in the accelerate slurm examples directory, but they have `activateEnvironment.sh` spelled correctly in the README instructions 🙃